plugins/k8s-rebase/test/test-skill.sh

```
@@ -0,0 +1,1310 @@
+import json, os
+p = '$repo/.claude/settings.json'
+d = json.load(open(p))
+d.pop('worktree', None)
+if d: json.dump(d, open(p, 'w'), indent=2)
+else: os.remove(p)
+" 2>/dev/null
+    fi
+    error "Launch failed for $(repo_short "$repo")"; return 1
+  fi
+  # Append session ID to running file for reliable stop
+  local _sid=$(cat "$RESULTS_DIR/.session_id_$_repo_key" 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)